pkg/v1/tkg/web/src/app/views/landing/wizard/shared/components/widgets/cluster-class-info/cluster-class-info.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, Input, OnInit } from '@angular/core';
+import clusterClassData from './cluster-class-data.json';
+
+@Component({
+    selector: 'app-cluster-class-info',
+    templateUrl: './cluster-class-info.component.html',
+    styleUrls: ['./cluster-class-info.component.scss']
+})
+export class ClusterClassInfoComponent implements OnInit {
+    @Input() providerType: string;
+
+    clusterClasses: any = {};
```

Comment:
It would be worth defining a clusterClass object type/interface instead of using 'any'